### PR TITLE
Bump Xcode from 15.4 to 16.0

### PR DESCRIPTION
**Pull request by uhooi**
### Overall

- https://x.com/the_uhooi/status/1806761341061103857
- #233

### Xcode 16

- https://developer.apple.com/documentation/xcode-release-notes/xcode-16-release-notes

#### View

- https://x.com/the_uhooi/status/1828987235632586810
- https://developer.apple.com/documentation/swiftui/view?changes=latest_minor

#### Retroactive

- https://x.com/stzn3/status/1803918835185246609
- https://forums.swift.org/t/accepted-se-0364-warning-for-retroactive-conformances-of-external-types/65015
- https://github.com/swiftlang/swift-evolution/blob/main/proposals/0364-retroactive-conformance-warning.md

#### count(where:)

- https://x.com/natpanferova/status/1819269134276088105
- https://nilcoalescing.com/blog/CountTheNumberOfObjectsThatPassATestInSwift/
- https://github.com/swiftlang/swift-evolution/blob/main/proposals/0220-count-where.md
- https://developer.apple.com/documentation/swift/int/words-swift.struct/count%28where%3A%29

### EditorConfig

- https://editorconfig.org/
- https://github.com/pinterest/ktlint/blob/abb24b946cb2cc35037640dbb9b91850cd7c0070/.editorconfig
- https://github.com/rust-lang/rust/blob/6c3485512fc95fa9c43cb831607bfec9d2153b6e/.editorconfig

### CODE REVIEW

File: LokiPackage/Package.swift [hunk 1]
```diff
@@ -1,19 +1,7 @@
...
-    static let algorithms: Self = .product(name: "Algorithms", package: "swift-algorithms")
-    static let logdogUI: Self = .product(name: "LogdogUI", package: "Logdog")
-    static let playbook: Self = .product(name: "Playbook", package: "playbook-ios")
-    static let playbookUI: Self = .product(name: "PlaybookUI", package: "playbook-ios")
-}
-
-private extension PackageDescription.Target.PluginUsage {
-    static let licenses: Self = .plugin(name: "LicensesPlugin", package: "LicensesPlugin")
-}
-
```
Comment (ojun9) [inline LokiPackage/Package.swift:16]: `static let`の定義を継続させたい場合は、以下のようなextensionを生やすことで回避できます！

```swift
extension PackageDescription.Target.Dependency: @unchecked @retroactive Sendable {}
extension PackageDescription.Target.PluginUsage: @unchecked @retroactive Sendable {}
```

Comment (uhooi) [reply LokiPackage/Package.swift:16]: 確かにそれでいけそう！ ありがと〜

Comment (uhooi) [reply LokiPackage/Package.swift:16]: とはいえ Sendable チェックはできる限り外したくないので悩みどころ、、

Comment (uhooi) [reply LokiPackage/Package.swift:16]: 継続しないことにした！ :pray:

**Pull request merged**